Makefile

```diff
@@ -52,6 +52,11 @@ init-container:
 		$(DEFAULT_RUN_ARGS) \
 		$(HARBOR)/$(IMAGE):$(VERSION) -c '/var/www/html/install-plugins.sh'
 
+clean-plugins:
+	@docker run --name=$(PROJECT_NAME) -p 127.0.0.1:80:80/tcp \
+		$(DEFAULT_RUN_ARGS) \
+		$(HARBOR)/$(IMAGE):$(VERSION) -c 'rm -rf /var/www/html/modules/* /var/www/html/themes/*'
+
```

Comment:
@nomadicoder What does this do?  This would only happen if you ran the command locally, so is there an equivalent command in the Dockerfile?

Reply:
This cleans out the installed plugins on demand in the mounted volume, so there's no equivalent in the Dockerfile.

Reply:
@nomadicoder I guess I just want to make sure this isn't something that needs to happen in our Rancher instance because this is only a local command.

Reply:
@sensei100 It only runs locally, but to be sure, I can make it operate on the local volume directory.

Reply:
@sensei100 Updated to only be able to operate on the local plugin directories.